### PR TITLE
Added the cluster fields

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [x] documentation
- [ ] dependencies

## Description

Add the cluster fields:
- CreationDate
- StoppedDate
- ClusterState
- ClusterName

### CODE REVIEW

File: cloud_governance/policy/aws/monitor/cluster_run.py [hunk 1]
```diff
@@ -26,27 +37,39 @@ def run_policy_operations(self):
...
+                        cluster_data[cluster_tag]['ClusterState'] = instance_state
+                        cluster_data[cluster_tag]['StoppedDate'] = stopped_date_time
                     cluster_data[cluster_tag]['Instances'].append(instance_data)
                     cluster_data[cluster_tag]['InstanceCount'] = len(cluster_data[cluster_tag]['Instances'])
                     cluster_data[cluster_tag]['Stopped'] = int(cluster_data[cluster_tag]['Stopped']) + stopped_instances
                     cluster_data[cluster_tag]['Running'] = int(cluster_data[cluster_tag]['Running']) + running_instances
                 else:
                     cluster_data[cluster_tag] = {
+                        'ClusterName': cluster_tag.split('/')[-1].upper(),
+                        'ClusterName2': cluster_tag.split('/')[-1].lower(),
```
Comment (ebattat) [inline cloud_governance/policy/aws/monitor/cluster_run.py:72]: @athiruma, do u think we need the lower also ?

Comment (athiruma) [reply cloud_governance/policy/aws/monitor/cluster_run.py:72]: I added it on safe side.

**Comment by athiruma** [comment]
@ebattat ??

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**